ci_framework/roles/libvirt_manager/tasks/virsh_checks.yml

```diff
@@ -0,0 +1,39 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Start and enable libvirt service
+  become: true
+  ansible.builtin.service:
+    name: libvirtd
+    state: started
+    enabled: yes
+
+- name: Get libvirt group users
+  ansible.builtin.getent:
+    database: group
+    key: libvirt
+  register: libvirt_group_users
+
+- name: Add user if not exists in libvirt group
+  when: "'cifmw_libvirt_manager_user' not in ansible_facts.getent_group['libvirt']"
+  become: true
+  ansible.builtin.user:
+    name: "{{ cifmw_libvirt_manager_user }}"
+    groups: libvirt
+    append: true
+
+- name: Check virsh version
+  ansible.builtin.command: virsh --connect qemu:///system --version
```

Comment:
IMHO this one will fail, since, while the user has been added to the group just before, it requires a re-authentication in order to make it available in the user env. Group modification isn't "live", unfortunately.

Reply:
I've deleted it.